src/partials/welcome.html

```diff
@@ -205,4 +205,10 @@ <h2 class="instructions" translate>welcome.ALREADY_CONNECTED</h2>
         </div>
 
     </div>
+
+    <a ng-if="!ctrl.showLoadingIndicator && ctrl.lastConnectedClientOS === 'ios'" class="desktop-beta-button" href="https://threema.ch/download-md">
```

Comment:
When scanning the code, the badge remains visible for about half a second. Could probably be improved by better positioning (as discussed).

Reply:
Amended!